tests/gluon_autolog/test_gluon_autolog.py

```diff
@@ -0,0 +1,73 @@
+import random
+import warnings
+
+import mxnet.ndarray as nd
+import numpy as np
+import pytest
+from mxnet.gluon import Trainer
+from mxnet.gluon.contrib.estimator import estimator
+from mxnet.gluon.data import Dataset, DataLoader
+from mxnet.gluon.loss import SoftmaxCrossEntropyLoss
+from mxnet.gluon.nn import HybridSequential, Dense
+from mxnet.metric import Accuracy
+
+import mlflow
+import mlflow.gluon
+
+client = mlflow.tracking.MlflowClient()
+
+
+class LogsDataset(Dataset):
+    def __init__(self):
+        self.len = 1000
+
+    def __getitem__(self, idx):
+        return nd.array(np.random.rand(1, 32)), nd.full(1, random.randint(0, 10), dtype="float32")
+
+    def __len__(self):
+        return self.len
+
+
+@pytest.fixture
+def gluon_random_data_run():
+    mlflow.gluon.autolog()
+
+    with mlflow.start_run() as run:
+        data = DataLoader(LogsDataset(), batch_size=128, last_batch="discard")
+
+        model = HybridSequential()
+        model.add(Dense(64, activation="relu"))
+        model.add(Dense(64, activation="relu"))
+        model.add(Dense(10))
+        model.initialize()
+        model.hybridize()
+        trainer = Trainer(model.collect_params(), "adam",
+                          optimizer_params={"learning_rate": .001, "epsilon": 1e-07})
+        est = estimator.Estimator(net=model, loss=SoftmaxCrossEntropyLoss(),
+                                  metrics=Accuracy(), trainer=trainer)
+
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
+            est.fit(data, epochs=5)
+
+    return client.get_run(run.info.run_id)
+
+
+@pytest.mark.large
+def test_gluon_autolog_logs_expected_data(gluon_random_data_run):
+    data = gluon_random_data_run.data
+    assert "train accuracy" in data.metrics
+    assert "train softmaxcrossentropyloss" in data.metrics
```

Comment:
Could we also add validation metrics to this test case?

Reply:
I'll look into this. I expect to be able to add it.